app/pulp/app/serializers/progress.py

```diff
@@ -0,0 +1,42 @@
+from rest_framework import serializers
+
+from pulp.app import models
+from pulp.app.serializers import ModelSerializer
+
+
+class ProgressReportSerializer(ModelSerializer):
+
+    message = serializers.CharField(
+        help_text="The message shown to the user for the progress report.",
```

Comment:
That's a good question, but I don't know the answer to it. DRF's docs scarcely mention anything to do with internationalization or localization, so I expect this is the place to do it unless we want to get really fancy and do a lot of work to support localizing responses per-request.